src/main.js

```diff
@@ -3,13 +3,21 @@
 import Vue from 'vue';
 import VueLogger from 'vuejs-logger';
 import Vuetify from 'vuetify';
+import VueSanitize from 'vue-sanitize';
 import App from './App';
 
 require('../node_modules/vuetify/dist/vuetify.min.css');
 
 Vue.config.productionTip = false;
 Vue.use(VueLogger);
 Vue.use(Vuetify);
+Vue.use(VueSanitize, {
+  allowedTags: ['a'],
+  allowedAttributes: {
+    // eslint-disable-next-line
```

Comment:
This is minor, but it's safer to disable specific rules only:
```suggestion
    // eslint-disable-next-line quote-props
```

Or if we'd like to disable it per block, we can do it like this:
```
/* eslint-disable quote-props */
Vue.use(VueSanitize, {
  allowedTags: ['a'],
  allowedAttributes: {
    'a': ['data-url'],
  },
});
/* eslint-enable quote-props */
```